### ISSUE TITLE
Import bypass using lambda Caesar Cipher Shift

### ISSUE THREAD

**Issue by discriminating** [mache102/puso#10]
![Image](https://cdn.discordapp.com/attachments/953021220662706196/1097239519922749450/image.png)

You can bypass the import verboser by simply using a lambda function to decypher cyphered python code, then running it through the eval() builtin method.

The code to encrypt a message:
```py
# Lambda function to encrypt a string using the Caesar cipher with a shift of 3
encrypt = lambda s: ''.join(chr((ord(c) + 3 - 32) % 95 + 32) if ' ' <= c <= '~' else c for c in s)

encrypted = encrypt("__import__('os').system('ls')")
print(f"decrypt = lambda s: eval(''.join(chr((ord(c) - 3 - 32) % 95 + 32) if ' ' <= c <= '~' else c for c in s)); decrypt('{encrypted}');")
```

Then, just append that onto the same line and you can import files.

**Comment by mybearworld** [comment]
Disallowing `exec` and `eval` entirely, if possible, would probably be the best thing to do fix this and #9.

**Comment by mache102** [comment]
thx; disallowed those two and a `compile` thingy.
